docs/reference/release-notes/8.13.0.asciidoc

```diff
@@ -28,6 +28,12 @@ If your cluster is running on ECK 2.12.1 and above, this may cause problems with
 To resolve this issue, perform a rolling restart on the non-master-eligible nodes once all Elasticsearch nodes
 are upgraded.
 
+* Due to a memory leak in pytorch 2.1.2, the `pytorch_inference` process running on Machine Learning nodes can consume
```

Comment:
> Due to a memory leak in pytorch 2.1.2

We don't know with absolute certainty that it's pytorch 2.1.2 (it could be MKL), do we still want to state this?